src/components/transactionList.tsx

```diff
@@ -51,10 +51,26 @@ const TransactionList = ({ txs }) => {
       ),
     },
     {
-      title: 'Value',
-      dataIndex: 'value',
-      key: 'value',
-      render: props => props.color !== undefined && <TokenValue {...props} />,
+      title: 'Outputs',
+      dataIndex: 'outputs',
+      key: 'outputs',
+      render: outputs => (
+        <List
+          itemLayout="vertical"
+          split={false}
+          dataSource={outputs}
+          renderItem={output => (
+            <List.Item>
+              Address:{' '}
+              <Link to={`/explorer/address/${output.address}`}>
+                {output.address}
```

Comment:
I suggest to use `shortenHex` on the address here, to save some horizontal space on narrow screens.

Reply:
Done